src/ex4.py

```diff
@@ -0,0 +1,32 @@
+#Surprise!
+def ex4(month : int) -> str:
+    month_len = ""
+    match month:
+        case 1:
+            month_len = "31 days"
+        case 2:
+            month_len = "28/29 days"
+        case 3:
+            month_len = "31 days"
+        case 4:
+            month_len = "30 days"
+        case 5:
+            month_len = "31 days"
+        case 6:
+            month_len = "30 days"
+        case 7:
+            month_len = "31 days"
+        case 8:
+            month_len = "31 days"
+        case 9:
+            month_len = "30 days"
+        case 10:
+            month_len = "31 days"
+        case 11:
+            month_len = "30 days"
+        case 12:
+            month_len = "31 days"
+        case _:
+            month_len = "Invalid input! Please enter month number between 1-12"
+    print(month_len)
+    return month_len
```

Comment:
👎 

potresti non usare la variabile month_len ma scrivere direttamente `return` al posto di `month_len = `, inoltre, potresti raggruppare i cases, se due cases hanno entrambi "31 days" inutile scrivere `return 31 days` due volte, potresti scrivere
```python
  ....
  case 10:
  case 12:
     return "31 days"
```

(penso che in python si possa fare)

Reply:
Non si può fare in sequenza, ma si può usare l'operatore | .
Sistemo subito